### ISSUE TITLE
Namespaces are not supported

### ISSUE THREAD

**Issue by thomastvedt** [dotansimha/graphql-code-generator#992]
**Describe the bug**
I get an error "Namespaces are not supported" when trying to run yarn gql-gen after upgrading to v 0.14.5.
I'm using "react-scripts-ts": "3.1.0", that is create react app "version 1" with TypeScript support using TSC (not babel). I know that namespaces are not supported if we use babel to transpile, but I'm pretty sure my project (react-scripts-ts) uses TSC and not babel. Is the new version of graphql-code-generator using babel in some way? To analyze .tsx-files when looking for gql tags?

package.json devDependencies:
```
    "graphql-code-generator": "^0.14.5",
    "graphql-codegen-typescript-client": "^0.14.5",
    "graphql-codegen-typescript-common": "^0.14.5",
    "graphql-codegen-typescript-react-apollo": "^0.14.5",
    "graphql-codegen-typescript-resolvers": "^0.14.5",
    "graphql-codegen-typescript-server": "^0.14.5",
```

codegen.yml:
```
schema:
- "http://localhost:8080/graphql"
documents:
- "./src/**/*.+(tsx|ts|graphql)"
config: {}
generates:
  ./src/models/:
    config:
      scalars:
        Date: Date
    plugins:
    - "typescript-common"
require: []

```
Output from command:
```
  $ yarn gql-gen
  yarn run v1.9.4
  $ /Users/thomastvedt/projects/myproject/node_modules/.bin/gql-gen
   ✔ Parse configuration
   ❯ Generate outputs
     ❯ Generate ./src/models/
       ✔ Load GraphQL schemas
       ✖ Load GraphQL documents
         →   6 |   export const midRadius = 214.41; // radius of middle circle
         Generate


     We found 1 errors

     Failed to generate ./src/models/
SyntaxError: /Users/thomastvedt/projects/myproject/.tsx: Namespaces are not supported.
1 | import * as moment from 'moment';
2 | 
> 3 | export namespace WheelConstants {
```

**To Reproduce**
1. Create React App version 1, using wmonks react-scripts-ts
2. Dont upgrade to CRA 2.0 with TypeScript support (I want to keep namespace support ++ too much to rewrite)
3. Run the following command: `gql-gen`

**Expected behavior**
I expected to be able to generate TypeScript code with namespaces without errors from babel

**Environment:**
 - OS: osX
 - Codegen: v0.14.5
 - Node: v10.8.0

**Comment by dotansimha** [comment]
@DAB0mB I think you can help with that :)

**Comment by kamilkisiela** [comment]
@DAB0mB Reproduction https://github.com/dotansimha/graphql-code-generator/pull/1019

It works with namespaces but not when some contain `gql` tag.

**Comment by DAB0mB** [comment]
Will check that soon.

**Comment by dotansimha** [comment]
@DAB0mB I think you can use the typescript ast parser instead of babel-typescript...

**Comment by DAB0mB** [comment]
It works when I run it directly through `graphql-tag-pluck` package. Still checking and if necessary will us TS compiler

**Comment by dotansimha** [comment]
Fixed and waiting for release. @thomastvedt you can use the latest alpha (`0.16.0-alpha.22d89977`).
Thanks @kamalgill @DAB0mB .

**Comment by dotansimha** [comment]
Fixed in 0.15.2 :)
